app/pages/donation-stripe.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div>
+    <SubTitle title="Lahmacun donation" :maintitle="true" />
+    <div class="container my-8">
+      <div v-if="$fetchState.pending" class="center">
+        Loading...
+      </div>
+
+      <div v-if="donationContent" class="max-w-4xl mx-auto">
+        <div class="mb-4">
+          <h2>{{ donationContent.title.rendered }}</h2>
+        </div>
+        <div class="grid md:grid-cols-2 md:gap-8 gap-4">
+          <div>
+            <div v-sanitize="[sanitizeOptions, donationContent.content.rendered]" />
+          </div>
+
+          <div>
+            <form action="https://cms.lahmacun.hu/wp-json/stripe/return_checkout_session_listener" method="GET">
+              <div>
+                <p class="mb-2">
+                  <strong>
+                    You can choose from the following options:
+                  </strong>
+                </p>
+                <div class="my-2">
+                  <input type="radio" id="option1" name="is_recurring" value="no" checked />
```

Comment:
Could we style it a bit more Lahma-like? The blue seems to me mismatchy

Reply:
OK, good idea!